prescient_sdk/config.py

```diff
@@ -0,0 +1,32 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    """
+    Default configuration for the Prescient SDK.
+
+    Configuration is handled using [pydantic-settings](https://docs.pydantic.dev/latest/concepts/pydantic_settings/)
+
+    Order of precedence for configuration values:
+
+    1. Environment variables are always highest precedence and will override any other configuration values
+    2. `.env` file: if a `.env` file is present in the root of the project, it will be used
+    """
+
+    model_config = SettingsConfigDict(
+        env_file=".env", env_file_encoding="utf-8", env_prefix="PRESCIENT_"
+    )
+
+    endpoint_url: str = "https://enexus.server-uat.prescient.earth"
```

Comment:
Because this is for many customer, we don't want defaults here. An exception should also be thrown if they are not set.